desktop/analytics/account_creation.js

```diff
@@ -69,6 +69,18 @@ if (Cookies.get('analytics-signup')) {
   }
 }
 
+// Created account on the Inquire via Phone modal (via email)
+analyticsHooks.on('mediator:auth:sign_up:success', ({context, user}) => {
+  if (context === 'show phone number') {
+    trackAccountCreation({
+      signup_service: 'email',
+      email: user.email,
+      user_id: user.id,
+      context: context
+    })
+  }
+})
+
```

Comment:
We already have a function for account creation tracking, but since it only gets invoked on an actual form submission (after signup, it should show the phone numbers without page refresh or form submission), I needed to add this observer to `'mediator:auth:sign_up:success'`. Ideally, these two functions should be consolidated as well.